index.html

```diff
@@ -123,11 +123,7 @@ <h4>Multi-Post Stories Gain+Glory</h4>
               <li>JavaScript</li>
               <li>html</li>
             </ul>
-            <button class="btn">
-              <a href="https://github.com/yonas44/space-tourism-website"
-                >See Project</a
-              >
-            </button>
+            <button onclick="displayModal(1)" class="btn">See Project</button>
```

Comment:
- [ ] When I click on `see project` button, I see completely different project and images. Please make sure that Popup window data corresponds to the one where users clicked `see project`